client/app/bundles/shared/components/Heading.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react';
+import css from './Heading.scss';
+
+type Props = {
+  large?: boolean,
```

Comment:
Should all these fields be mandatory rather than optional?

Reply:
I think mandatory as then user can use whatever he requires based on props passed. @nshki What are your views on this ?

Reply:
I think it makes sense to make the `label` prop mandatory, but the rest can stay optional.

Reply:
I don't know how to make them optional. What do I have to do ?

Reply:
You used `?` so that should be enough :)

Reply:
Okay :)